src/backend/distributed/executor/multi_real_time_executor.c

```diff
@@ -660,9 +674,16 @@ ManageTaskExecution(Task *task, TaskExecution *taskExecution,
 			int32 connectionId = connectionIdArray[currentIndex];
 			int32 fileDesc = fileDescriptorArray[currentIndex];
 			int closed = -1;
+			uint64 returnBytesReceived = 0;
```

Comment:
Minor: Maybe `returnBytesReceived` -> `bytesReceived`. The `return` makes more sense within `MultiClientCopyData` where it is like a return value.